app/Repositories/MessageRepository.php

```diff
@@ -120,6 +120,12 @@ public function updateMessagesForContest($contest, $messages)
                 $data['type'] = $type;
                 $data['key'] = $key;
 
+                if ($data['type'] === 'leaderboard') {
+                    if (! array_key_exists('show_images', $data)) {
+                        $data['show_images'] = 0;
```

Comment:
The other option is to do something like this, but i didn't like the hidden field thing http://nielson.io/2014/02/handling-checkbox-input-in-laravel/